### PR TITLE
Enable Rolling and Galactic CI

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
Enables CI for Rolling and Galactic distros. Will fix https://github.com/ros-controls/ros2_controllers/pull/200 from the repos file once it gets merged. The extra repos and CI files for rolling and galactic will be removed once ros2_control and ros2_controllers are released for rolling and galactic and synced (for main repos).

### CODE REVIEW

**Comment by vatanaksoytezer** [comment]
All the CI runs should pass. The failing ones are failing because of the flaky OMPL test... Shall we disable that as well @henningkayser ? Since this is in my sprint right now, I will work on enabling them real quick

**Comment by vatanaksoytezer** [comment]
See #495

**Comment by henningkayser** [comment]
Please squash all your small fixup commits and do a rebase, then lgtm

**Comment by vatanaksoytezer** [comment]
> Please squash all your small fixup commits and do a rebase, then lgtm

Great! Done. Should be mergable if this CI passes

File: moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp [hunk 1]
```diff
@@ -38,7 +38,8 @@
 #include <moveit/moveit_cpp/moveit_cpp.h>
 #include <moveit/planning_scene_monitor/current_state_monitor.h>
 #include <moveit/common_planning_interface_objects/common_objects.h>
-
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <geometry_msgs/msg/quaternion_stamped.h>
```
Comment (henningkayser) [inline moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp:42]:
```suggestion
#include <geometry_msgs/msg/quaternion_stamped.hpp>
```

File: moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp [hunk 2]
```diff
@@ -67,7 +68,7 @@ MoveItCpp::MoveItCpp(const rclcpp::Node::SharedPtr& node, const Options& options
   if (!loadPlanningSceneMonitor(options.planning_scene_monitor_options))
   {
     const std::string error = "Unable to configure planning scene monitor";
-    RCLCPP_FATAL(LOGGER, error);
+    RCLCPP_FATAL_STREAM(LOGGER, error);
```
Comment (henningkayser) [inline moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp:71]: I don't really understand. What's wrong with not using stream?

File: moveit_ros/planning/trajectory_execution_manager/CMakeLists.txt [hunk 3]
```diff
@@ -1,5 +1,7 @@
 set(MOVEIT_LIB_NAME moveit_trajectory_execution_manager)
 
+find_package(tinyxml_vendor)
```
Comment (henningkayser) [inline moveit_ros/planning/trajectory_execution_manager/CMakeLists.txt:3]:
```suggestion
find_package(tinyxml_vendor REQUIRED)
```

File: README.md [hunk 4]
```diff
@@ -6,9 +6,9 @@ The MoveIt Motion Planning Framework for **ROS 2**. For ROS 1, see [MoveIt 1](ht
 
 ## Continuous Integration Status
 
-
 [![Formatting (pre-commit)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml?query=branch%3Amain)
 [![Build and Test (Foxy)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test.yaml?query=branch%3Amain)
+[![Build and Test (Rolling)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test_rolling.yaml?query=branch%3Amain)
```
Comment (henningkayser) [inline README.md:11]: Could you add a Galactic badge as well?

File: .github/workflows/build_and_test_galactic.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```
Comment (henningkayser) [inline .github/workflows/build_and_test_galactic.yaml:1]: Right now these files include a lot of duplication. Not a big deal but we should probably consolidate these in the future.

File: moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp (hunk 2 above)
Comment (vatanaksoytezer) [reply moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp:71]: Rolling doesnt like char * with print style I can keep that style with %s though if you want. Stream solves this without needing any additional formatting

File: .github/workflows/build_and_test_galactic.yaml (hunk 5 above)
Comment (vatanaksoytezer) [reply .github/workflows/build_and_test_galactic.yaml:1]: I can probably clean them up and merge files.

Comment (vatanaksoytezer) [reply .github/workflows/build_and_test_galactic.yaml:1]: But if I merge them in one file (build_and_test.yaml would contain foxy, rolling and galactic). We won't be able to create different badges.

**Review by henningkayser — APPROVED** [review]

**Pull request merged**